utilities-core/src/main/java/uk/gov/justice/services/yaml/ObjectMapperFactory.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.services.yaml;
+
+import uk.gov.justice.services.common.converter.jackson.ObjectMapperProducer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+public class ObjectMapperFactory {
```

Comment:
This Class name should be clear about what this class does...maybe...YamlObjectMapperFactory